src/Source/Potree2Source.js

```diff
@@ -0,0 +1,126 @@
+     * * __`boundingBox`__ - Contains the minimum and maximum of the axis aligned bounding box. This bounding box is cubic and aligned to fit to the octree root.
+     * * __`tightBoundingBox`__ - This bounding box thightly fits the point data (optional).
+     * * __`spacing`__ - The minimum distance between points at root level.
+     * ```
+     * {
+     *     version: '2.0',
+     *     name: "sample",
+     *     description: "",
+     *     points: 534909153,
+     *     projection: "",
+     *     hierarchy: {
+     *         firstChunkSize: 1276,
+     *         stepSize: 4,
+     *         depth: 16
+     *     },
+     *     offset: {
+     *         0: 1339072.07,
+     *         1: 7238866.339,
+     *         2: 85.281
+     *     },
+     *     scale: {
+     *         0: 0.001,
+     *         1: 0.001,
+     *         2: 0.002
+     *     },
+     *     spacing: 24.476062500005355,
+     *     boundingBox: {
+     *         min: {
+     *             0: 1339072.07,
+     *             1: 7238866.339,
+     *             2: 85.281
+     *         },
+     *         max: {
+     *             0: 1342205.0060000008,
+     *             1: 7241999.275,
+     *             2: 3218.2170000006854
+     *         }
+     *     },
+     *     tightBoundingBox: {
+     *         min: {
+     *             0: 1339072.07,
+     *             1: 7238866.339,
+     *             2: 85.281
+     *         },
+     *         max: {
+     *             0: 1342205.0060000008,
+     *             1: 7241999.275,
+     *             2: 3218.2170000006854
+     *         }
+     *     },
+     *     encoding: "BROTLI",
+     *     attributes: [
+     *         0: {
+     *             name: "position",
+     *             ...
+     *         },
+     *         1: {
+     *             name: "intensity",
+     *             ...
+     *         },
+     *         2: {
+     *             name: "classification",
+     *             ...
+     *         },
+     *         3: {
+     *             name: "user data",
+     *             ...
+     *         },
+     *         4: {
+     *             name: "rgb",
+     *             ...
+     *         },
+     *         ...
+     *     ]
+     * }
+     * ```
+     *
+     * @extends Source
+     *
+     * @constructor
+     */
+    constructor(source) {
+        if (!source.file) {
+            throw new Error('New Potree2Source: file is required');
+        }
+
+        super(source);
+        this.file = source.file;
+        this.fetcher = Fetcher.arrayBuffer;
+        this.extensionOctree = 'octree.bin';
+
+        this.whenReady = (source.cloud ? Promise.resolve(source.cloud) : Fetcher.json(`${this.url}/${this.file}`, this.networkOptions))
+            .then((cloud) => {
+                this.pointAttributes = cloud.attributes;
+                this.baseurl = `${this.url}`;
+                this.extension = 'bin';
+                this.parse = Potree2BinParser.parse;
```

Comment:
In other sources of itowns (except `PotreeSource` it seems...), this property is named `parser`, could you please rename it?
```suggestion
                this.parser = Potree2BinParser.parse;
```

Reply:
Ok